summerofcode/2023.md

```diff
@@ -242,6 +242,24 @@ If you are a project maintainer and consider mentoring during the GSoC 2023 cycl
 - Difficulty: Hard
 - Upstream Issue (URL): https://github.com/keptn/integrations/issues/19
 
+#### Build Standard KeptnTaskDefinition Library
+- Description: [Keptn Lifecycle Toolkit (KLT)](https://github.com/keptn/lifecycle-toolkit) can run pre and post deployment tasks. Currently KLT supports the [Deno](https://deno.land) runtime. KeptnTasks can be used for almost anything that an individual can dream up and script in JavaScript. However, users (especially new adopters) usually need a useful set of example tasks that they can use and / or modify to suit their requirements.
+
+This project would build such a library of re-usable tasks for common tasks such as:
+
+- Notifying Chat tools
+- Triggering external tools (eg. a pipeline)
+- Integrations with Systems of Record (i.e. to update a CMDB)
+
+This project lends itself to GSoC due to the modular nature of the tasks which almost guarantees that something will be delivered - the task delivery can be planned an organised easily into sprints across the project lifecycle. It is also relatively easy - given that the tasks are simply JavaScript - it should be easy for most first time contributors.
+
+- Expected outcome: A catalogue of useful, re-usable `KeptnTaskDefinitions` that will serve as useful reference material for adopters
+- Recommended Skills: Typescript and / or JavaScript, basic understanding of Keptn Lifecycle Toolkit, Kubernetes basic knowledge, knowledge of CRDs
+- Mentor[s]: [Adam Gardner (adam at agardner dot net)](https://github.com/agardnerit)
```

Comment:
@agardnerIT Can you ask one of the people listed in https://github.com/cncf/foundation/blob/main/project-maintainers.csv#L512 to LGTM this PR? Your name is not in that file and CNCF org admins require the "blessing" of the maintainers from that list.

Reply:
Hey @aliok, I spoke with @agardnerIT and LGTM :)